.github/workflows/test-plugin.yml

```diff
@@ -97,3 +97,88 @@ jobs:
     - name: Run CPU backend examples
       working-directory: triton/third_party/triton_shared/python/examples
       run: pytest .
+
+
+
+  build_and_test_triton_shared_arm:
```

Comment:
once you have this nailed down, can we refactor and resuse the same steps as above?

Reply:
absolutely, this is a pretty hacky workaround but I believe with llvm build 66886578, all of this should be easy to get rid of, also I've been working on lowering linalg.matmul to ArmSVE, so really been enjoying some of that work